app/javascript/pages/listings/for-rent.tsx

```diff
@@ -85,9 +84,9 @@ const getRentalHeader = (
           {t("rentalDirectory.findMatchingListings")}
         </LinkButton>
       </p>
-      <Markdown className="text-base">
-        {t("rentalDirectory.orGetHelpCalculating", { incomeLink: getHelpCalculatingIncomeLink() })}
-      </Markdown>
+      <a className="text-base text-primary-dark" href={getHelpCalculatingIncomeLink()}>
```

Comment:
Any consequences that I am missing with removing Mardown functionality and just using a normal html tag?